lib/security/WSSecurity.js

```diff
@@ -3,10 +3,11 @@
 var crypto = require('crypto')
   , passwordDigest = require('../utils').passwordDigest;
 
-function WSSecurity(username, password, passwordType) {
+function WSSecurity(username, password, options) {
```

Comment:
As the `passwordType` String param was not only replaced by an `options` Object parameter,  we should do something like this when setting `this._passwordType`:
```javascript
options = options || {};

this._passwordType = typeof options === 'string' ? options : options.passwordType;
this._hasTimeStamp = options.hasTimeStamp;
```

Also, I think we should update the README in order to let the people know, that an `options` Object is expected and what it could contain.